### DISCUSSION TITLE
12/05 다음 주 글 초안 공유하기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#44] [동글동글 3기 피드백]
다음 주 글 주제나 정리 중인 초안을 공유해 주세요! (주제와 소제목 정도 수준으로)
다른 사람들은 어떤 글을 어떻게 구성하고 있는지 보면서 시야를 넓혀나가 볼까요? 👀

**Comment by mh0223** [comment]
현재 자바스크립트의 실행컨텍스트에 관해서 첫번째 글을 썼는데, 이어서 더 자세한 내용으로 이어서 쓸 예정입니다.

**Comment by hansol-woff** [comment]
three.js 혹은 리액트 배우고 있는 내용을 조금 처음 시작하는 것부터 차근차근 써볼까 생각하고 있습니다..!

**Comment by ksj0621** [comment]
깃헙 블로그 만드는  과정에서 여러 오류들을 겪어서 저와 같은 분들께 참고가 되었으면 하는 바람에 제가 겪은 오류들을 정리해서 글을 써볼 예정입니다

**Comment by baaanjy** [comment]
프로젝트 기초 세팅과 관련한 내용을 정리해보려고 합니다.(패키지 설치, 라이브러리 적용, 익스텐션, 깃 규칙 등)

**Comment by Devkdy22** [comment]
프로젝트에 사용할 리액트에 관련해서 기초부터 공부하며 작성 해볼까 생각 중입니다..
